content/integrate/redis-data-integration/architecture.md

```diff
@@ -136,14 +136,28 @@ traffic. The diagram below shows this configuration:
 
 {{< image filename="images/rdi/ingest/ingest-active-passive-vms.png" >}}
 
+See [Install on VMs]({{< relref "/integrate/redis-data-integration/installation/install-vm" >}})
+for more information.
+
 ### RDI on Kubernetes
 
-You can run RDI in a namespace on your own Kubernetes cluster with
-a separate deployment for each of the control plane
-components (REST API, operator, and metrics exporter). The operator
-creates and configures deployments for the collector and stream processor
-when you start a pipeline from the CLI tool. This means that you must
-provide the operator with a service account that has permissions to create and manipulate pods in the namespace. 
+You can use the RDI [Helm chart](https://helm.sh/docs/topics/charts/) to install
+on [Kubernetes (K8s)](https://kubernetes.io/). This creates:
```

Comment:
it's good to explicitly mention Openshift as most customers will use it
something like on Kubernetes (K8s) including Red Hat Openshift